INSTALL.md

```diff
@@ -52,18 +50,28 @@ This will use `/opt/circleci` as the base install location.
 
 First, set one of these variables as appropriate for for your installation target.
 
+#### For Linux AMD64
```

Comment:
AMD64 caused a bit of confusion with one customer, as x86_64 is the more common term (I think?) could we include both in this title?

Reply:
Happy to use x86_64 everywhere